packages/insomnia-app/app/ui/redux/modules/global.js

```diff
@@ -138,6 +145,38 @@ export function newCommand(command, args) {
           },
         });
         break;
+      case COMMAND_PLUGIN_THEME:
+        const parsedTheme = JSON.parse(decodeURIComponent(args.theme));
+        showModal(AskModal, {
+          title: 'Import Theme',
```

Comment:
```suggestion
          title: 'Install Theme',
```

Potentially? We use this wording in the generator on the website.